components/Nav/Navbar/index.tsx

```diff
@@ -24,9 +24,11 @@ const NavBar: React.FC<{
             )}
         >
             <NavBarContent setShowOnboardModal={setShowOnboardModal} />
-            <style>{`
-                background-position-x:
-            `}</style>
+            <style>
+                {`
+                    background-position-x:
```

Comment:
Should we keep this?

Interesting use of style tags in react components

Reply:
they shouldnt be in there, was just because we were using tailwind